style.css

```diff
@@ -0,0 +1,683 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  max-width: 100%;
+}
+
+body {
+  max-width: 100%;
+  min-height: 100vh;
+  background-color: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+header {
+  background-color: #fff;
+}
+
+.amburger {
+  display: flex;
+  flex-direction: column;
+}
+
+.bar {
+  display: block;
+  width: 33px;
+  height: 3px;
+  margin: 2px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #162ae3;
+}
+
+.nav-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem;
+}
+
+section {
+  padding: 5rem 1.25rem 6rem;
+  background-color: #fff;
+  margin-bottom: 20%;
+}
+
+.major-container {
+  display: flex;
+  flex-direction: column;
+  border-bottom-left-radius: 80px;
+  background-image: url("images/footer/Header bg.png");
```

Comment:
- [ ] Great job adding the background image 👏. However, it's not displaying correctly compared to the original design. Please fix that.😊

### Your design 
![Screenshot from 2023-02-17 17-42-09](https://user-images.githubusercontent.com/89228743/219731132-7b17ffa7-f3aa-4a6a-9bdc-6907478adfac.png)
### The original design
![Screenshot from 2023-02-17 17-42-41](https://user-images.githubusercontent.com/89228743/219731339-bd09b2d4-5198-419e-a3ab-1581ba3e4cb4.png)

### To help you  follow these steps: :point_down: 

1. Use the background image on the mobile version of the Figma design. You can export from there or send a message on slack or on Zoom I will send it to you.
2. Instead of putting it to the `major-container` `div`, put it to the `section`.
3. After the change the `major-container` `div`, and the `section` style may look like this
```
section {
  padding: 5rem 1.25rem 6rem;
  background-color: #fff;
  background-image: url('images/footer/Header bg.png');
  background-repeat: no-repeat;
  background-position: inherit;
  background-size: 100% 100%;
  margin-bottom: 20%;
}
```
```
.major-container {
  display: flex;
  flex-direction: column;
  border-bottom-left-radius: 80px;
  padding-top: 70px;
  justify-content: center;
}
```
### Note:
Change the background image with the one on the mobile version.